src/Anonymization/Anonymizator.php

```diff
@@ -235,22 +358,33 @@ protected function anonymizeTablePerColumn(string $table, array $anonymizers): \
             $table = $anonymizer->getTableName();
             $target = $anonymizer->getColumnName();
 
-            yield \sprintf('   - anonymizing %d/%d: "%s"."%s"...', $count, $total, $table, $target);
+            $this->output->write(\sprintf(
+                '- anonymizing %d/%d: "%s"."%s"...', $count, $total, $table, $target
```

Comment:
The output interface `write()` method could be the same as the logger, `write(message, string[] args)` in order to get rid of all those `sprintf()` calls in here ?

Reply:
@pounard I would like to suggest that we simply use `\sprintf()` in background and document that the message must be compatible with the `\sprintf()` format. The signature also would be `write(string $message, mixed ...$args)`. Seems good to you?